packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -820,15 +814,17 @@ class _FlutterPlatform extends PlatformPlugin {
     return processManager.start(command, environment: environment);
   }
 
-  void _pipeStandardStreamsToConsole(
+  Future<void> _pipeStandardStreamsToConsole(
```

Comment:
We're not doing anything with the result of this future at the callsite, so is it worth making this return a future?